### PR TITLE
Revised scope section, reordered, clarified language

### PR DESCRIPTION
**Pull request by mlagally**
Revised scope section - changed the order, clarifications provided.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-charter-drafts/pull/88.html" title="Last updated on Mar 23, 2023, 10:41 AM UTC (46841ce)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-charter-drafts/88/15443be...46841ce.html" title="Last updated on Mar 23, 2023, 10:41 AM UTC (46841ce)">Diff</a>

### CODE REVIEW

File: wot-wg-2023-draft.html [hunk 1]
```diff
@@ -210,38 +210,50 @@ <h2>Scope</h2>
...
           </dd>
           <dt>
-            <strong>Increase Descriptiveness of TDs</strong>:
+            	<strong>Increase Descriptiveness of TDs</strong>:
           </dt>
           <dd>
-            The WG will work on increasing the descriptive power of TDs to allow using them in more use cases. This includes
-            describing dynamic resources, historical values (timeseries), repeating payloads, both static and dynamic
+            The WG will work on increasing the descriptive power of TDs to allow using them in wider use cases. This includes
+            describing dynamic resources, historical values (timeseries), describing payloads, both static and dynamic
```
Comment (egekorkan) [inline wot-wg-2023-draft.html:247]: we are already describing payloads so I would prefer to keep the previous wording

File: wot-wg-2023-draft.html [hunk 2]
```diff
@@ -210,38 +210,50 @@ <h2>Scope</h2>
...
         <dl>
-          <dt><strong>Update Existing Specifications</strong>:</dt>
-          <dd>The WG will add new features and address issues discovered during implementation of already published specifications. 
-            Unlike the last charter, these would not be limited to backward-compatible updates.
+          <dt>
+		<strong>Support New Use Cases</strong>:
+          </dt>
+          <dd>Address the requirements of new use cases, such as Digital Twins, scalable events, 
+	    and interworking with edge and cloud systems.
+	    The WG intends to define canonical TDs, add a mechanism to sign TDs,
```
Comment (egekorkan) [inline wot-wg-2023-draft.html:218]: I do not understand why signing etc. are moved here.

File: wot-wg-2023-draft.html [hunk 3]
```diff
@@ -210,38 +210,50 @@ <h2>Scope</h2>
       <p>During this charter, the Working Group would work on the following:</p>
 
         <dl>
-          <dt><strong>Update Existing Specifications</strong>:</dt>
-          <dd>The WG will add new features and address issues discovered during implementation of already published specifications. 
-            Unlike the last charter, these would not be limited to backward-compatible updates.
+          <dt>
+		<strong>Support New Use Cases</strong>:
+          </dt>
+          <dd>Address the requirements of new use cases, such as Digital Twins, scalable events,
```
Comment (egekorkan) [inline wot-wg-2023-draft.html:216]: We have talked way too many times about the notion of scalable events. There is still no agreement on what is really meant here. If we are talking about cloud events, it is already proposed for payload binding.

**Review by egekorkan — CHANGES_REQUESTED** [review]
@mlagally I think that there a bit too many changes with different kind of things addressed. It would be easier to split this PR so that moving the content around and adding new content are clearly indicated.

File: wot-wg-2023-draft.html (hunk 1 above)
Comment (mlagally) [reply wot-wg-2023-draft.html:247]: There was a problem with describing event payloads - we discussed in the Profile TF some time ago.
I don't understand the use cases (and proposed solution) for repeating payloads.

File: wot-wg-2023-draft.html (hunk 2 above)
Comment (egekorkan) [reply wot-wg-2023-draft.html:218]: And it is part of the charter already. I do not understand how signing canonical TDs are part of a use case discussion that is comparable to digital twins. As the current charter says: 

> In other specifications, the WG intends to add a mechanism to sign TD

Comment (mmccool) [reply wot-wg-2023-draft.html:218]: Could move signing etc. under improving security, but I don't think it belongs under "new use cases".  It doesn't really address any specific requirement of a new use case, it improves security overall.

**Comment by mmccool** [comment]
- I am generally ok with reordering to put new use cases at the top.   I am also ok with most of the new use cases mentioned (including scalable events and cloud and edge integration)
- However, I think the signing part should be moved to "Improve Security and Privacy", not new use cases
- "better describing payloads" IMO is a good general statement, and appropriate at this level.  Being able to support "repeating" payloads is one way to better describe them, but there are other improvements possible.  As was discussed before, "repeating payloads" is too confusing and undefined for someone without knowledge of the rest of the discussion in the TD calls (and I'm thinking of the AC reviewers here)

In short, while I think this PR needs some work, it adds some useful items.   I agree though that it would be easier to break into smaller parts.

**Comment by egekorkan** [comment]
TD Call of 15.03:
- better describing payloads: We think that this is a better way (less specific but correct).

File: wot-wg-2023-draft.html (hunk 3 above)
Comment (mmccool) [reply wot-wg-2023-draft.html:216]: I think we can add more use cases, but "scalable events" is a feature, not a use case.  I would be ok with adding "such as Digital Twins and interworking with edge and cloud systems."

I think that "scalable events" are a work item that relates to "Cloud interworking".  However, "integration" might be a better word in general than "interworking".
```suggestion
          <dd>Address the requirements of new use cases, such as Digital Twins and integration with edge and cloud systems.
```

File: wot-wg-2023-draft.html [hunk 4]
```diff
@@ -210,38 +210,50 @@ <h2>Scope</h2>
...
 
         <dl>
-          <dt><strong>Update Existing Specifications</strong>:</dt>
-          <dd>The WG will add new features and address issues discovered during implementation of already published specifications. 
-            Unlike the last charter, these would not be limited to backward-compatible updates.
+          <dt>
+		<strong>Support New Use Cases</strong>:
+          </dt>
+          <dd>Address the requirements of new use cases, such as Digital Twins, scalable events, 
+	    and interworking with edge and cloud systems.
```
Comment (mmccool) [inline wot-wg-2023-draft.html:217]: to go along with suggestion above
```suggestion
```

File: wot-wg-2023-draft.html (hunk 1 above)
Comment (mmccool) [reply wot-wg-2023-draft.html:247]: Agree with Ege.  There are some good ideas here, but mixed in with other things that I don't agree with, and it's too hard to address the things that need fixing with suggested changes.

**Comment by mmccool** [comment]
Meeting March 23:
- Group edits to this PR during mtg to resolve issues

**Comment by mmccool** [comment]
Please review this new version.
